server/s3_copy_object_action.cc

```diff
@@ -428,6 +428,7 @@ void S3CopyObjectAction::save_metadata() {
   new_object_metadata->set_content_type(
       source_object_metadata->get_content_type());
   new_object_metadata->set_md5(motr_writer->get_content_md5());
+  new_object_metadata->setacl(auth_acl);
```

Comment:
is it possible to have
new_object_metadata->setacl(request->get_default_acl()) and do away with auth_acl member variable